pebblo/app/service/loader/document/document.py

```diff
@@ -18,11 +18,11 @@ def _get_or_create_document(self, doc, data_source):
         logger.debug("Create or update AIDocument")
         filter_query = {
             "appId": self.app_name,  # loadId or AppId ( Doubt)
-            "sourcePath": doc.get("source_path"),
+            "loaderSourcePath": doc.get("source_path"),
```

Comment:
This need to be `sourcePath` as we would fetch document based on appId and sourcePath.